biothings/hub/databuild/backend.py

```diff
@@ -157,23 +157,27 @@ def get_src_metadata(self):
 
             if src and src.get("download"):
                 # Store the latest success dump time
-                if src["download"]["status"] == "success":
-                    src_meta.setdefault(src["_id"], {}).setdefault("download_date", src["download"]["started_at"])
+                src_meta.setdefault(src["_id"], {})
+                src_meta[src["_id"]].setdefault(
+                    "download_date", src["download"].get("last_success")
+                )
```

Comment:
@sengineer0 I think we should still keep the old way as the fallback (get the value from `src["download"]["started_at"]` if a successful one). The reason is some existing data sources don't have `last_success` field, and they may not be updated any more to add `last_success` field.

Reply:
so, you want to use `src["download"]["started_at"]` from the the last succesful dump/download, in the case the `last_success` is None or not exists, right?
sure, I will create another PR to update it.